pydoctor/epydoc2stan.py

```diff
@@ -119,31 +134,46 @@ def look_for_intersphinx(self, name: str) -> Optional[str]:
         return self.obj.system.intersphinx.getLink(name)
 
     def link_to(self, identifier: str, label: "Flattenable") -> Tag:
+        # :Raises _EpydocLinker.LookupFailed: If the identifier cannot be resolved and self.strict is True.
```

Comment:
Maybe have this as docstrig instead of comment?

Reply:
Will do

Reply:
Actually, I remember. I did not si that not to override the docstirng inherited by `ParsedDocstring`. 
Since this an implementation detail because we're always using the `_CachedEpydocLinker`, and I did not want to change the pretty well documented docstring that we already have for `link_to` and `link_xref`.